src/QAEmployee.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+
+public class QAEmployee extends Employee {
+	private int qaGeneralLevel;
```

Comment:
add some validation here

Reply:
I added the validation for this attribute.